cmd/flowrunner/testdata/flows/all_actions_test.json

```diff
@@ -204,59 +203,99 @@
                         ],
                         "type": "contact_groups_removed"
                     },
-                    "step_uuid": "229bd432-dac7-4a3f-ba91-c48ad8c50e6b"
+                    "step_uuid": "8f570cd6-588b-4b10-acdc-e181352d44cc"
                 },
                 {
                     "action_uuid": "f01d693b-2af2-49fb-9e38-146eb00937e9",
                     "event": {
-                        "contacts": [
-                            {
-                                "name": "Ben Haggerty",
-                                "uuid": "ba96bf7f-bc2a-4873-a7c7-254d1927c4e3"
-                            }
-                        ],
                         "created_on": "2000-01-01T00:00:00.000000000-00:00",
-                        "text": "Hi Ben Haggerty, are you ready to complete today's survey?",
-                        "type": "broadcast_created"
+                        "msg": {
+                            "channel": {
+                                "name": "Android Channel",
+                                "uuid": "57f1078f-88aa-46f4-a59a-948a5739c03d"
+                            },
+                            "text": "Hi Ben Haggerty, are you ready to complete today's survey?",
+                            "urn": "tel:+12065551212",
+                            "uuid": "19ebde80-3969-47d3-a09e-6806aab9f510"
+                        },
+                        "type": "msg_created"
                     },
-                    "step_uuid": "229bd432-dac7-4a3f-ba91-c48ad8c50e6b"
+                    "step_uuid": "8f570cd6-588b-4b10-acdc-e181352d44cc"
                 },
                 {
                     "action_uuid": "d98c1e02-69df-4f95-8b89-8587a57ae0c3",
                     "event": {
                         "created_on": "2000-01-01T00:00:00.000000000-00:00",
-                        "text": "This is a message to each of Ben Haggerty's urns.",
-                        "type": "broadcast_created",
-                        "urns": [
-                            "tel:+12065551212",
-                            "facebook:1122334455667788",
-                            "mailto:ben@macklemore",
```

Comment:
note that there is no msg_created event for this URN because we can't send to it